Fixed section link to latest released version

svick opened the pull request.

livarcocc commented on README.md:
```diff
@@ -61,7 +61,7 @@ You can download .NET Core SDK as either an installer (MSI, PKG) or a zip (zip,
 
 > **Note:** please be aware that below installers are the **latest bits**. If you 
-> want to install the latest released versions, please check out the [section above](#download-links).)
+> want to install the latest released versions, please check out the [section above](#looking-for-v1-of-the-net-core-tooling).
```
It seems like you are missing a closing parenthesis here, which is breaking formatting for this file.

svick replied:
That was intentional, there is no opening parenthesis, so there shouldn't be a closing one either.

I do see the table broken [here](https://github.com/svick/cli/blob/7ebee29dc1a231dee0e53fc44a6cc23112e945b6/README.md), but [it looks broken in the previous version too](https://github.com/svick/cli/blob/09b6be599007ea4d791dc4b7bbfa63541e819419/README.md) (which looks fine on the main page of the repo), so I don't think it's related to the parenthesis.

svick replied:
I think the rendering issue could instead be caused by [GitHub's switch to a new Markdown parser](https://githubengineering.com/a-formal-spec-for-github-markdown/). I wasn't able to figure out how to fix it (at least not without switching from link references `[text][link-name]` to normal links `[text](http://whatever)`).

am11 replied:
Line 67 has an extra `:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|`, removing that fixes the rending problem.

svick replied:
@am11 You're right. Thanks, fixed.

am11 replied:
@svick, no problem and thank you for sharing the link. :)
I think the new parser is hardened than the old one; as previously it was ignoring the extra column in the md table header based on it's absence in the rest of the rows.

livarcocc commented:
@dotnet-bot Test Windows_NT x86 Debug Build

svick commented:
Since the table issue has been fixed by https://github.com/dotnet/cli/pull/6084, which also prevented my table fix to be merged, I have removed the table-fixing commit and rebased on top of current master.

livarcocc commented:
@dotnet-bot Test Fedora24 x64 Release Build
@dotnet-bot Test OpenSUSE42.1 x64 Debug Build
@dotnet-bot Test Ubuntu16.10 x64 Debug Build
